pyproject.toml

```diff
@@ -8,7 +8,7 @@ readme = "README.md"
 packages = [{include = "cultcargo"}]
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = "^3.9 <3.13"
```

Comment:
This looks odd to me. `">=3.9, <3.13"` may be clearer.

Reply:
I actually thought "^" was the replacement for ">=" all this time!

Reply:
They have slightly different meanings. So-called caret requirements can be a bit weird: https://python-poetry.org/docs/dependency-specification/#caret-requirements.